### PR TITLE
update some connectors to use YAML specs

### LINKED ISSUE #11891: Update 5 connectors to use YAML spec
Full context in parent issue

Make a PR to any connector (e.g. `source-stripe`) bumping its CDK dependency and changing its spec to be a YAML, then publish a new version of that connector

┆Issue is synchronized with this [Monday item](https://airbyte-bunch.monday.com/boards/2536787439/pulses/2537067784) by [Unito](https://www.unito.io)

Suggested connectors to migrate: 

1. stripe
2. salesforce
3. hubspot
4. google sheets
5. ??

### PR DESCRIPTION
**Pull request by pedroslopez**
## What
#11668 introduced yaml spec support, now let's actually use it!

## How
Update 5 connectors to use a `spec.yaml`:

- [x] stripe
- [x] salesforce
- [x] hubspot
- [x] google sheets
- [x] exchange-rates

I basically ran the old `spec.json` files through https://codebeautify.org/json-to-yaml and made minor tweaks (mainly change single quotes to double quotes)

Also updated some references to the old files and bumped connector versions.
Doc updates coming soon w/ references to new `spec.yaml` files.

### CODE REVIEW

File: airbyte-integrations/connectors/source-stripe/setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     author_email="contact@airbyte.io",
     packages=find_packages(),
     install_requires=MAIN_REQUIREMENTS,
-    package_data={"": ["*.json", "schemas/*.json", "schemas/shared/*.json"]},
+    package_data={"": ["*.json", "*.yaml", "schemas/*.json", "schemas/shared/*.json"]},
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-stripe/setup.py:22]: note: _not_ adding *.yaml here weirdly didn't seem to cause any issues 🤔

File: airbyte-integrations/connectors/source-stripe/source_stripe/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/stripe
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-stripe/source_stripe/spec.yaml:3]: feels a _little_ odd to be doing this in a yaml file, but I think it's still right?

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            - client_id
+            - client_secret
+            - refresh_token
+            - credentials_title
+          properties:
+            credentials_title:
+              type: string
+              title: Credentials Title
+              description: Name of the credentials set
+              const: OAuth Credentials
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml:37]: This is slightly different than the `spec.json` counterpart: previously `default` and `enum` were defined, but it looks like that's not necessary anymore and was causing test failures.

File: airbyte-integrations/connectors/source-hubspot/setup.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1.49",
+    "airbyte-cdk~=0.1.55",
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-hubspot/setup.py:9]: Just pointing out that I noticed only hubspot has a patch version in the requirements. Others either don't have a version at all, or are set to `~=1`

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml (hunk 3 above)
Comment (sherifnada) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml:37]: Yup! Thanks for the change

**Review by sherifnada — APPROVED** [review]

**Comment by pedroslopez** [comment]
/publish connector=connectors/source-google-sheets auto-bump-version=false

**Comment by pedroslopez** [comment]
/publish connector=connectors/source-exchange-rates auto-bump-version=false

**Pull request merged**